src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1624,6 +1659,9 @@ class ProjectTransformer(SingletonTransformer):
     def _entity_id(self) -> api.UUID4:
         return self._api_project.document_id
 
+    def entity_dates(self) -> Callable:
+        return lambda: self._entity(self._api_project)
```

Comment:
Again, confused by the nullary `lambda:` If we stick with returning a Calalble here (and elsewhere), it needs parameters for the argument and return types. Consider creating a variable or protocol class to store the parameterized annotation s you don't need to repeat it.